packages/ui/src/components/IconButton.tsx

```diff
@@ -1,18 +1,30 @@
 import { PropsWithChildren } from 'react';
 
 import { Button } from '../components/Button';
-import { ColorTokens, SizeTokens, ThemeTokens, useTheme } from '../core';
+import {
+  ColorTokens,
+  RadiusTokens,
+  SizeTokens,
+  ThemeTokens,
+  useTheme,
+} from '../core';
 
 export function IconButton({
   children,
   onPress,
   size = '$s',
   color = '$primaryText',
+  backgroundColor = '$background',
```

Comment:
I think it'd make more sense to create a separate component for the floating action button rather than introduce this extra config here.